### ISSUE TITLE
`rand` doesn't accept dims as tuple. 

### ISSUE THREAD

**Issue by rofinn** [JuliaStats/Distributions.jl#678]
`rand` doesn't seem to support passing in dimensions as a tuple anymore. I'm not sure if this was an intentional decision that I've missed, but this behaviour doesn't match base.

Example)
```julia
julia> rand(Normal(0, 0.01), (10, 100))
ERROR: MethodError: Cannot `convert` an object of type Tuple{Int64,Int64} to an object of type Array{Float64,1}
This may have arisen from a call to the constructor Array{Float64,1}(...),
since type constructors fall back to convert methods.
Stacktrace:
 [1] rand(::Distributions.Normal{Float64}, ::Tuple{Int64,Int64}) at /Users/rory/repos/Boltzmann.jl/.playground/packages/v0.6/Distributions/src/univariates.jl:183
```
vs 
```julia
julia> rand((10, 100))
10×100 Array{Float64,2}:
 0.791279  0.223855  0.0777301   0.184161  0.130529  0.240192   …  0.365646  0.514122  0.0208268  0.249949  0.227289   0.554622
 0.4493    0.14642   0.199054    0.593586  0.397336  0.973833      0.938297  0.719154  0.488888   0.652405  0.668898   0.540961
 0.868193  0.834226  0.619585    0.787526  0.341186  0.848141      0.897664  0.650697  0.305104   0.369852  0.03108    0.5799
 0.205769  0.791125  0.00804583  0.618595  0.324247  0.0800603     0.240182  0.151047  0.0108451  0.368934  0.467204   0.83309
 0.536467  0.668956  0.787667    0.54657   0.197809  0.0099607     0.985709  0.444187  0.0450553  0.596016  0.104536   0.973567
 0.689792  0.511078  0.912089    0.162852  0.400569  0.638219   …  0.750985  0.916195  0.393101   0.76599   0.393456   0.349624
 0.433147  0.715756  0.152019    0.324823  0.850758  0.19169       0.900873  0.343492  0.333474   0.468986  0.675624   0.713998
 0.730337  0.312889  0.344235    0.217639  0.471501  0.93017       0.908684  0.728918  0.454474   0.90631   0.166854   0.471954
 0.110002  0.488152  0.520586    0.354729  0.459721  0.396132      0.541623  0.955917  0.528852   0.926     0.0826705  0.134977
 0.680024  0.726517  0.795166    0.397505  0.593282  0.590878      0.748384  0.354363  0.95401    0.135797  0.132613   0.305414
```

**Comment by jbaker92** [comment]
Also experiencing this

**Comment by ararslan** [comment]
I don't know why this would have been intentional, and even if it was, +1 for changing it back.

**Comment by richardreeve** [comment]
@ararslan This is now fixed by #830 and can be closed:
```julia
julia> rand(Normal(0, 0.01), (10, 100))
10×100 Array{Float64,2}:
  0.0315121    -0.0230304    -0.00042996  …  -0.0137607    -0.0197865
  0.0103462    -0.00120799    0.0131326      -0.00856801    0.00435454
 -0.00821726    0.00176663    0.0027254       0.00816941   -0.00511772
  0.0120402     0.00227967   -0.0129321      -0.00727854   -0.000132097
  0.00811648   -0.0179516     0.002416       -0.000212831  -0.0226197
 -0.000258034  -0.0100507     0.0065923   …   0.00480116   -0.00285501
  0.00130505    0.00137829   -0.0121277       0.00492566    0.00358066
 -0.00337088    0.00589859   -0.00328614      0.0113277     0.0144824
  0.00946772    0.0072907     0.00956231      0.000653386   0.00670413
  0.0126367    -0.000565205   0.0181225       0.0145836     0.0022595
```
